src/tasks/AndroidAppBuilder/Templates/monodroid.c

```diff
@@ -226,13 +226,13 @@ mono_droid_runtime_init (const char* executable, int managed_argc, char* managed
     appctx_values[1] = bundle_path;
 
     char *file_name = RUNTIMECONFIG_BIN_FILE;
-    int str_len = strlen (bundle_path) + strlen (file_name) + 1;
-    char *file_path = (char *)malloc (sizeof (char) * (str_len + 1));
-    int res1 = snprintf (file_path, str_len, "%s/%s", bundle_path, file_name);
+    int str_len = strlen (bundle_path) + strlen (file_name) + 2;
```

Comment:
I didn't see this change before, but I actually don't like this. I think previously it was clear where the extra `+1`s were coming from, and you should maybe add a comment clarifying that if you're going to just use `+2` here. It's a bit too magicky for me otherwise. This can be adjusted in the followup PR, so not a big concern.

Reply:
I was trying the avoid the calculation afterwards.

Reply:
You avoid an extra `add reg, 1` instruction but this is an essentially trivial cost (and quite likely free given the slack I'd assume is present in the pipeline). Readability is much more of a concern—getting buffer lengths correct in C is hard, and so the more obviously correct and well documented constants like this are the better.

Reply:
There are about 3 extra adds. But I agree that it was better self-documented with the previous version. I was debating myself, while I was changing it. For an internal test, I guess that startup time could be sacrificed a little bit here.

Reply:
Definitely not three? Looks like just an add and a mov: https://godbolt.org/z/zbYTGjM3x

Either way, I guarantee the timer we're using isn't even precise enough to notice the difference, if there is any in practice.

Reply:
It is line 230, 231 and 235. And the link doesn't seem to open successfully. I could change it. `+1` is definitely more reader friendly.